bower.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "ember-infinity",
   "dependencies": {
-    "ember": "1.13.7",
+    "ember": "1.13.9",
+    "ember-data": "1.13.9",
```

Comment:
I would keep these in alphabetical order, so that subsequent upgrades don't churn bower.json excessively